docs/core/diagnostics/runtime-garbage-collection-events.md

```diff
@@ -0,0 +1,500 @@
+---
+title: "Garbage Collection Runtime Events"
+description: View detailed information about garbage collection events.
```

Comment:
Need to make sure we have SEO information in here in the description. Also, the title should be sentence cased.